doc/man1/flux-jobs.adoc

```diff
@@ -0,0 +1,100 @@
+// flux-help-include: true
+FLUX-JOBS(1)
+==============
+:doctype: manpage
+
+
+NAME
+----
+flux-jobs - list jobs submitted to Flux
+
+
+SYNOPSIS
+--------
+*flux* *jobs* ['OPTIONS']
+
+
+DESCRIPTION
+-----------
+flux-jobs(1) is used to list jobs run under Flux.  By default only
+pending and running jobs for the current user are listed.  Additional
+jobs and information can be listed using options listed below.
+
+
+OPTIONS
+-------
+*-a*::
+List all jobs of the current user, including completed jobs is in the
+inactive state.  Equivalent to specifying
+'--state=pending,running,inactive'.
+
+*-A*::
+List all jobs from all users, including completed jobs is in the inactive state.
+Equivalent to specifying '--state=pending,running,inactive --userid=all'.
+
+*--suppress-header*::
+For default output, do not output column headers.
+
+*-u, --userid*'=UID'::
```

Comment:
We may want to reserve the `-u` short arg for username.   I suspect users will prefer to do `flux jobs -u herbein1` (or `flux jobs -u $USER`) than `flux jobs -u $(id -u herbein1)`.

Reply:
good idea, perhaps it should just be `--user` and take username or userid?  i.e. assume userid if it's only numbers.

Reply:
actually, `--user` is in `squeue`, which actually takes multiple usernames.  Perhaps could be done with `flux-jobs`, but that's for another day.  I can add that to the issue list.

Reply:
hmmmm, I currently support `--userid=all`, but I probably shouldn't support `--user=all`, b/c a username could be named `all`.  Hmmm.  `--user=0` means all?  not the most obvious.

Reply:
Well, it seems `lsf` just assumes there will never be a user named `all`.

`-u user_name | -u user_group | -u all`